azext_iot/digitaltwins/_help.py

```diff
@@ -140,6 +140,112 @@ def load_digitaltwins_help():
             az dt reset -n {instance_name}
     """
 
+    helps["dt data-history"] = """
+        type: group
+        short-summary: Manage and configure data history connections.
+    """
+
+    helps["dt data-history create"] = """
+        type: group
+        short-summary: Creates a data history connection between a Digital Twins instance and another resource.
```

Comment:
"Create data history connections between a Digital Twins instance and supported resources." 

The 'create adx' command description has a specific resource.